lib/bot.js

```diff
@@ -135,16 +136,18 @@ class Bot {
     logger.debug('Channel Mapping', channelName, this.channelMapping[channelName]);
     if (ircChannel) {
       const username = author.username;
-      const colorIndex = (username.charCodeAt(0) + username.length) % NICK_COLORS.length;
-      const coloredUsername = irc.colors.wrap(NICK_COLORS[colorIndex], username);
       let text = this.parseText(message);
+      let displayUsername = username;
+      if (this.ircNickColor) {
+        displayUsername = irc.colors.wrap(NICK_COLORS[(username.charCodeAt(0) + username.length) % NICK_COLORS.length], username);
```

Comment:
Since we have lodash available you could use `_.sample(NICK_COLORS)`: https://lodash.com/docs#sample